src/server/game/Entities/Player/Player.cpp

```diff
@@ -984,15 +984,15 @@ Player::~Player()
 }
 
 void Player::CleanupsBeforeDelete(bool finalCleanup)
-{ 
+{
```

Comment:
@IntelligentQuantum wow... big diff here ^_^ where are the actual changes?

Reply:
It happened because of this file
https://github.com/azerothcore/azerothcore-wotlk/blob/master/.editorconfig#L7

Reply:
actual changes
![Screenshot_20200328_174318](https://user-images.githubusercontent.com/14080516/77823937-62de9a00-711c-11ea-8b1d-dc4c5386c9e4.png)